codecov.yml

```diff
@@ -14,14 +14,16 @@ coverage:
         threshold: 3%
         flags: 
           - backend
-    patch:
+        informational: false
```

Comment:
For future reference, changing `informational` to `false` here should cause Codecov status to fail for backend code when Go test coverage doesn't meet the Codecov target established here. 
However, this should not block PRs as long as Codecov statuses are not a required Github status check for merging into our main branch.